### PR TITLE
Issue 2:  Users can view their entries -Brandon Moy

### PR DESCRIPTION
**Pull request by brandon-moy**
[view entries demo.webm](https://user-images.githubusercontent.com/109567499/189968825-e9083de4-22ef-4adf-a4ff-47588b13c210.webm)

### CODE REVIEW

**Comment by brandon-moy** [comment]
Changed layout of pages to start on Entries page and clicking the Entries tab or NEW button will swap pages.

[view swapping entries new entry.webm](https://user-images.githubusercontent.com/109567499/189991068-4d899d15-f12a-4a8d-b2f8-dbb903809454.webm)

[Adding entries and page refresh.webm](https://user-images.githubusercontent.com/109567499/189991091-431409e1-71ce-4d62-945e-5f0c11667368.webm)

**Comment by brandon-moy** [comment]
Change made to `href ` value for the Entries view swap to show the Entries title and new entries button.

[Change to show Entries title.webm](https://user-images.githubusercontent.com/109567499/189992066-f152c252-4f85-41c6-ba9f-4398ff843a0e.webm)

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Does this show the last view the user was on when they re-open the page? I don't see any code here to show the last view the user was on when the page first loads.

**Comment by brandon-moy** [comment]
Added in the code to save the page view and reload to it.

[page reload demo.webm](https://user-images.githubusercontent.com/109567499/190013286-aeae5ca1-d096-49cb-8b0c-d7124b2a99b9.webm)

File: js/main.js [hunk 1]
```diff
@@ -19,7 +24,76 @@ function saveEntry(event) {
   data.entries.unshift(newEntry);
   $img.setAttribute('src', '../images/placeholder-image-square.jpg');
   $form.reset();
+
+  var addEntry = createEntryTree(newEntry);
+  $entryList.prepend(addEntry);
+  $noEntry.className = '.no-entry hidden';
```
Comment (codingcodymiller) [inline js/main.js:30]: I don't think you meant to have a `.` in the value here.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: js/main.js (hunk 1 above)
Comment (brandon-moy) [reply js/main.js]: Oops, must've gotten my mind switched up a little when making the DOM elements. Made the change!

image

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**